source/probability_theory_4_finite.Rmd

```diff
@@ -243,87 +243,66 @@ cards, half red and half black, and deal out five cards *without
 replacing them* after each card is dealt; this simulates the choice of
 five students from among the fifty.
 
-We can also work the example with random numbers. But if so, we cannot
-simply designate "01-25" as boys and "26-50" as girls (ignoring 51-00
-when they occur), because if we did so the probability of a girl would
-be the same on each pick. Instead, to simulate non-replacement, we
-specify that no number can be taken into the sample twice, just as no
-student can be chosen twice. That is, if we come across a number more
-than once, we simply ignore it after the first occurrence. More
-specifically:
+We can no longer use our procedure from before.  If we designated
+"1-25" as being girls and "26-50" as being boys and then proceeded to
+draw random numbers, the probability of a girl would be the same on
+each pick.
 
-**Step 1.** Let "01-25" = girls, "26-50" = boys
+At this point, it is important to note that---for this particular
+problem---we do not need to distinguish between particular girls (or
+boys). That is, it does not matter *which* girl (or boy) is selected
+in a given trial. Nor did we pay attention to the *order* in which we
+selected girls or boys. This is an instance of Case 4 discussed
+above. Subsequent problems will deal with situations where the order
+of selection, and the particular individuals, do matter.
 
-**Step 2.** Select five non-duplicating numbers; if a number comes up
-more than once, ignore it. Count whether there are four numbers "01-25,"
-and one "26-50." If so, write "yes," otherwise "no."
+Our approach then is to mimic having the class in front of us: an
+array of 50 strings, half of the entries 'boy' and the other half
+'girl'.  We then shuffle the class (the array), and choose the first N
```

Comment:
Thinking about it - how about `np.random.choice(..., replace=True)`?

Reply:
You mean replace=False?

Reply:
Sorry - yes I do mean 'replace=False'